src/system/Form/ToggleGroup.tsx

```diff
@@ -0,0 +1,72 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { ThemeUIStyleObject } from 'theme-ui';
+
+/**
+ * Internal dependencies
+ */
+import { Box } from '../Box';
+import { Button } from '../Button';
+
+export interface ToggleGroupProps {
+	value?: string;
+	onChange?: ( value: string ) => void;
+	options?: {
+		value: string;
+		label: string;
+	}[];
+	sx?: ThemeUIStyleObject;
+}
+
+const ToggleGroup = ( props: ToggleGroupProps ) => {
+	const { options, value, onChange } = props;
+
+	const handleClick = ( e: React.MouseEvent< HTMLButtonElement >, val: string ) => {
+		e.preventDefault();
+		onChange?.( val );
+	};
+
+	return (
+		<Box
+			role="group"
+			dir="ltr"
+			tabIndex={ 0 }
+			sx={ {
+				background: 'layer.3',
+				p: 1,
+				display: 'inline-flex',
+				gap: 1,
+				borderRadius: 1,
+			} }
+		>
+			{ options?.map( option => (
+				<Button
```

Comment:
I wonder if it would be better to use actual radio buttons here and style to suit? If we use buttons, we should indicate the selected choice using `aria-checked`. There's a pattern we can follow here https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/radio_role

Reply:
Come to think of it, this component starts seeming a lot like the existing [RadioBoxGroup](https://deploy-preview-415--vip-design-system-components.netlify.app/?path=/docs/radioboxgroup--docs#default) component. Perhaps we can update that component instead, by adding a variant which is styled like our ToggleGroup? 🤔

Reply:
To @t-wright's comment: Yes. This is why we ask people to start developing components here instead of on the VIP Dashboard, so we can evaluate current options vs. creating new ones 🙏 .

The structure of the radio-group is radio under the hood. You can build a variant of the RadioBoxGroup, like @t-wright said.